### PR TITLE
docs: add fugue to ecosystems

### PR DESCRIPTION
**Pull request by goodwanghan**
docs: add fugue to ecosystems

@cpcloud

### CODE REVIEW

File: docs/web/community/ecosystem.md [hunk 1]
```diff
@@ -57,3 +57,44 @@ This would output a dataframe that looks like:
...
+```python
+import pandas as pd
+
+# schema: *,pred:double
+def predict(df: pd.DataFrame) -> pd.DataFrame:
+    model = load_model("somefile")
+    return df.assign(pred=model.predict(df))
+
+import fugue_ibis
+from fugue import FugueWorkflow
```
Comment (cpcloud) [inline docs/web/community/ecosystem.md:82]:
```suggestion
import pandas as pd

import fugue_ibis
from fugue import FugueWorkflow


# schema: *,pred:double
def predict(df: pd.DataFrame) -> pd.DataFrame:
    model = load_model("somefile")
    return df.assign(pred=model.predict(df))

```

**Review by cpcloud — CHANGES_REQUESTED** [review]
Awesome! Just a quick suggestion to move the imports up. Other than that, this is good to go.

File: docs/web/community/ecosystem.md (hunk 1 above)
Comment (goodwanghan) [reply docs/web/community/ecosystem.md]: Sure, I have fixed this @cpcloud

**Review by cpcloud — APPROVED** [review]
@goodwanghan Can you squash to a single commit? Thanks! Looks great!

**Comment by goodwanghan** [comment]
Sure I will squash and merge.

Thank you!

**Comment by goodwanghan** [comment]
@cpcloud will you squash and merge? or I need to squash it first?

**Review by jreback — COMMENTED** [review]
personally i would remove the actual code sample from here & the sql_to_ibis and just provide the prose & links. The home pages are fully formed and likely kept up to date.

**Comment by cpcloud** [comment]
Merging on green! Thanks @goodwanghan!

**Pull request merged**